keyboards/handwired/jotlily60/info.json

```diff
@@ -0,0 +1,110 @@
+{
+    "manufacturer": "Jotix",
+    "keyboard_name": "jotlily60",
+    "maintainer" : "jotix",
+    "url": "https://github.com/qmk/qmk_firmware/tree/master/keyboards/handwired/jotlily60",
+    "usb": {
+        "vid": "0x4A4F",
+        "pid": "0x2332",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "tapping": {
+        "term": 175
+    },
```

Comment:
This is generally more of a user config option.
```suggestion
```

Reply:
thank you for spoting that